packages/utils-connector-tools/src/requester.ts

```diff
@@ -51,33 +52,29 @@ const identity = (buff: Buffer): Promise<Buffer> => {
     return Promise.resolve(Buffer.from(buff));
 };
 
-const defaults = {
-    encoding: null,
-    followRedirect: false,
+const defaults: RequestInit = {
+    compress: false,
     headers: {
         'Accept-Encoding': 'gzip, deflate, br',
         'Accept-Language': 'en-US,en;q=0.8,es;q=0.6,fr;q=0.4',
         'Cache-Control': 'no-cache',
-        DNT: 1,
+        DNT: '1',
         Pragma: 'no-cache',
         'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/74.0.3729.131 Safari/537.36'
```

Comment:
Maybe a good opportunity to update the UA? It's almost 30 versions behind.
Maybe this should be updated as part of the release although if it's worked this far maybe a `good first issue`?

Reply:
good call, will do.

Reply:
I gave a sewcond thought to this one, as we discussed releasing this change as a single (and easily reversible change) I think is better if I update the UA in a different PR, so that, if any reason we need to revert this one that change does not also get reverted.